tee-worker/core/rpc-client/src/direct_client.rs

```diff
@@ -215,6 +216,19 @@ impl DirectApi for DirectClient {
 		decode_from_rpc_response::<u32>(&response_str)
 	}
 
+	fn get_mrenclave(&self) -> Result<MrEnclave> {
+		let jsonrpc_call: String =
+			RpcRequest::compose_jsonrpc_call("get_mrenclave".to_string(), Default::default())?;
+
+		// Send json rpc call to ws server.
+		let response_str = self.get(&jsonrpc_call)?;
+
+		let mrenclave: MrEnclave = decode_from_rpc_response::<MrEnclave>(&response_str)?;
+
+		info!("[+] Got enclave: {:?}", mrenclave);
```

Comment:
I think this log might be too noisy.

Reply:
Can I use mrenclave.to_base58() or hex::encode(mrenclave)?

Reply:
I thought about getting rid of entire log entry but I see we do similar logging in other methods so it's fine.
I'll resolve this conversation 🙃 .